### DISCUSSION TITLE
Query Param Description

### DISCUSSION THREAD

**Discussion by syahnur197** [usebruno/bruno#2393] [Ideas]
It would be nice if we have the ability to write a description of the query param, form payload, etc in the bru lang just like in Postman we can describe what a parameter does.

I think putting comment on top of the param should be sufficient

```
get {
  url: {{url}}/api/regions
}

query {
  # filter region by area
  area: foo
}
```

**Comment by B-SRIKRISHNAN** [comment]
Would be great in switching between params during development
